Feature: Keyboard Shortcuts for Camera Live Feed

rithviknishad opened the pull request:
Fixes #2374 

- Adds support for shortcut keys in camera feed controls. [See Shortcuts](https://github.com/coronasafe/care_fe/issues/2374#issuecomment-1127768998)
- Adds tooltip that shows label and keyboard shortcut to trigger
![image](https://user-images.githubusercontent.com/25143503/168713819-20a1c215-4d48-4742-8d63-aec6d3ef06f6.png)

developedBySJ reviewed (changes requested):
Add keyboard shortcuts to patient feed as well

nihal467 commented:
@rithviknishad @gigincg test failed, the shortcuts are not working, and also give a hint like button so that the user can refer to the shortcut

rithviknishad commented:
- Added shortcut keys for patient feed.
- Added help menu that lists all shortcut keys (in both asset cam feed and patient feed)

![image](https://user-images.githubusercontent.com/25143503/169040603-acae41b9-3f3d-45ba-8c7b-2f4e3ae93104.png)

![image](https://user-images.githubusercontent.com/25143503/169040697-0e763ba1-59ca-4b71-a76e-e00325138141.png)

nihal467 commented:
test approved

gigincg commented on src/Components/Facility/ConsultationDetails.tsx:
```diff
@@ -171,7 +171,7 @@ export const ConsultationDetails = (props: any) => {
         <nav className="flex justify-between flex-wrap">
           <PageTitle
-            title="Patient Dashboard"
+            title="Patient Details"
```
@rithviknishad Was this unintended

rithviknishad replied:
It was unintended action.
After looking into: https://github.com/coronasafe/care_fe/pull/2381/commits/a34d7821b4b41f2e2f6705200e0015da0eab84df and https://github.com/coronasafe/care_fe/pull/2381/commits/845a268db201a9c3b6a83b9715ca9675be81ee95, seems like intended action was to remove that commit from this PR since it was not related to this, but didn't expect this would happen. My bad. Resolved again in #2754
